cURL/Endpoint Examples/JSON Payload/decrypted-pdf-with-permissions-password.sh

```diff
@@ -8,7 +8,22 @@ UPLOAD_ID=$(curl --location 'https://api.pdfrest.com/upload' \
 
 echo "File successfully uploaded with an ID of: $UPLOAD_ID"
 
-curl 'https://api.pdfrest.com/decrypted-pdf' \
+DECRYPTED_OUTPUT=$(curl 'https://api.pdfrest.com/decrypted-pdf' \
 --header 'Api-Key: xxxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx' \
 --header 'Content-Type: application/json' \
---data-raw "{ \"id\": \"$UPLOAD_ID\", \"current_open_password\": \"encrypted\", \"current_permissions_password\": \"restricted\"}" | jq -r '.'
+--data-raw "{ \"id\": \"$UPLOAD_ID\", \"current_open_password\": \"encrypted\", \"current_permissions_password\": \"restricted\"}")
+
+echo $DECRYPTED_OUTPUT | jq -r '.'
+
+# All files uploaded or generated are automatically deleted based on the 
+# File Retention Period as shown on https://pdfrest.com/pricing. 
+# For immediate deletion of files, particularly when sensitive data 
+# is involved, an explicit delete call can be made to the API.
+
+# The following code is an optional step to delete decrypted files from pdfRest servers.
+
+DECRYPTED_ID=$(jq -r 'outputId' <<< $DECRYPTED_OUTPUT)
```

Comment:
This produces a jq compile error.

it needs a '.' before "outputId"

DECRYPTED_ID=$(jq -r '.outputId' <<< $DECRYPTED_OUTPUT)

This will likely need to be changed in all of the curl samples